backend/README.md

```diff
@@ -24,6 +24,7 @@ If you followed the instructions above or are using a different OS run the follo
 2. Create a `.env` file in the `backend/` **sub**folder of the current directory, it should look something like this:
 ```ini
 SECRET_KEY=<see note below>
+SIGNING_KEY=<see note below>
```

Comment:
how is one supposed to generate a `SIGNING_KEY`?

Reply:
sorry forgot to edit the note below